content/en/blog/2021/upcoming-networking-changes/index.md

```diff
@@ -0,0 +1,71 @@
+description: Understanding the upcoming changes to Istio networking, how they may impact your cluster, and what action to take.
+publishdate: 2021-04-08
+attribution: "John Howard (Google)"
+---
+
+## Background
+
+While Kubernetes networking is customizable, a typical pod's network will look like this:
+
+{{< image width="50%" link="./pod.svg" caption="A pod's network" >}}
+
+An application may choose to bind to either the loopback interface `lo` (typically binding to `127.0.0.1`), or the pods network interface `eth0` (typically to the pod's IP), or both (typically binding to `0.0.0.0`).
+
+Binding to `lo` allows calls such as `curl localhost` to work from within the pod.
+Binding to `eth0` allows calls to the pod from other pods.
+
+Typically, an application will bind to both.
+However, applications which have internal logic, such as an admin interface may choose to bind to only `lo` to avoid access from other pods.
+Additionally, some applications, typically stateful applications, choose to bind only to `eth0`.
+
+## Current behavior
+
+In Istio prior to release 1.10, the Envoy proxy, running in the same pod as the application, binds to the `eth0` interface and redirects all inbound traffic to the `lo` interface.
+
+{{< image width="50%" link="./current.svg" caption="A pod's network with Istio today" >}}
+
+This has two important side effects that cause the behavior to differ from standard Kubernetes:
+
+* Applications binding only to `lo` will receive traffic from other pods, when otherwise this is not allowed.
+* Applications binding only to `eth0` will not receive traffic.
+
+Applications that bind to both interfaces (which is typical) will not be impacted.
+
+## Future behavior
+
+Starting with Istio 1.10, the networking behavior is changed to align with the standard behavior present in Kubernetes.
+
+{{< image width="50%" link="./planned.svg" caption="A pod's network with Istio in the future" >}}
+
+Here we can see that the proxy no longer redirects the traffic to the `lo` interface, but instead forwards it to the application on `eth0`.
+As a result, the standard behavior of Kubernetes is retained, but we still get all the benefits of Istio.
+This change allows Istio to get closer to its goal of being a drop-in transparent proxy that works with existing workloads with [zero configuration](/blog/2021/zero-config-istio/).
+Additionally, it avoids unintended exposure of applications binding only to `lo`.
+
+## Am I impacted?
+
+For new users, this change should only be an improvement.
+However, if you are an existing user, you may have come to depend on the old behavior, intentionally or accidentally.
+
+To help detect these situations, we have added a check to find pods that will be impacted.
+You can run the `istioctl experimental precheck` command to get a report of any pods binding to `lo` on a port exposed in a `Service`.
+This command is available in Istio 1.10+.
+Without action, these ports will no longer be accessible upon upgrade.
```

Comment:
```suggestion
**Without action, these ports will no longer be accessible upon upgrade.**
```

Suggest highlight this.

Reply:
I think it makes sense to say if the error (since you are changing the msg from warning to error) has occurred, you must use revision based upgrade to thoroughly test your application continues to work with the new control plane.